test/presentation/user_action/user_action_details_view_model_test.dart

```diff
@@ -51,27 +56,83 @@ void main() {
       final store = Store<AppState>(
         reducer,
         initialState: AppState.initialState().copyWith(
+          userActionListState: UserActionListSuccessState([mockUserAction(id: 'id')]),
           userActionUpdateState: UserActionUpdatedState(UserActionStatus.DONE),
         ),
       );
 
       // When
-      final viewModel = UserActionDetailsViewModel.create(store);
+      final viewModel = UserActionDetailsViewModel.create(store, 'id');
 
       // Then
       expect(viewModel.displayState, UserActionDetailsDisplayState.SHOW_SUCCESS);
     });
   });
 
+  test("when action is on time", () {
+    // Given
```

Comment:
"When action is on time ..... and what?" Qu'est-ce que fait le test?
Maybe "When action is on time should show active color and correct text"

Reply:
J'en parlais à Adrien, pour moi le `then` est implicite quand tu lis ce qu'il y a dans la méthode. Mais ok j'ajoute un truc